### PR TITLE
Translation of javascript chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586

Please review.

@MSakamaki, I translated javascript chapter.
if you already translated javascript chapter, My pull-req stop, Tell me.

### CODE REVIEW

File: src/content/ja/2019/javascript.md [hunk 1]
```diff
@@ -0,0 +1,433 @@
+---
+part_number: I
+chapter_number: 1
+title: JavaScript
+description: 2019年のWeb AlmanacのJavaScriptの章では、Web上でどれだけJavaScriptを使用しているか、圧縮、ライブラリとフレームワーク、読み込み、ソースマップを網羅しています。
+authors: [housseindjirdeh]
+reviewers: [obto, paulcalvano, mathiasbynens]
+translators: [javascript]
```
Comment (bazzadp) [inline src/content/ja/2019/javascript.md:8]:
```suggestion
translators: [ksakae]
```

File: src/content/ja/2019/javascript.md [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+          <td>Dojo</td>
+          <td class="numeric">0.70%</td>
+          <td class="numeric">0.62%</td>
+        </tr>
+      </tbody>
+    </table>
+	<figcaption>図11. デスクトップとモバイルでのトップ JavaScript ライブラリ</figcaption>
+</figure>
+
+これまでに作成された中で最も人気のあるJavaScriptライブラリである[jQuery](https://jquery.com/)は、デスクトップページの85.03％、モバイルページの83.46％で使用されています。[Fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API)や[querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector)など、多くのブラウザAPIやメソッドの出現により、ライブラリが提供する機能の多くがネイティブ形式に標準化されました。jQueryの人気は衰退しているように見えるかもしれませんが、なぜ今でもウェブの大部分で使われているのでしょうか？
```
Comment (bazzadp) [inline src/content/ja/2019/javascript.md:283]:
```suggestion
これまでに作成された中で最も人気のあるJavaScriptライブラリである[jQuery](https://jquery.com/)は、デスクトップページの85.03％、モバイルページの83.46％で使用されています。[Fetch](https://developer.mozilla.org/ja/docs/Web/API/Fetch_API)や[querySelector](https://developer.mozilla.org/ja/docs/Web/API/Document/querySelector)など、多くのブラウザAPIやメソッドの出現により、ライブラリが提供する機能の多くがネイティブ形式に標準化されました。jQueryの人気は衰退しているように見えるかもしれませんが、なぜ今でもウェブの大部分で使われているのでしょうか？
```

File: src/content/ja/2019/javascript.md [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+<script type="module" src="main.mjs"></script>
+```
+
+ページ上のスクリプトに`type="module'`を使用しているサイトはどれくらいあるでしょうか？
+
+<figure>
+   <a href="/static/images/2019/javascript/fig13.png">
+      <img src="/static/images/2019/javascript/fig13.png" alt="図13. type=moduleを利用しているサイトの割合。" aria-labelledby="fig13-caption" aria-describedby="fig13-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1409239029&format=interactive">
+   </a>
+	<div id="fig13-description" class="visually-hidden">Bar chart showing 0.6% of sites on desktop use 'type=module', and 0.8% of sites on mobile.</div>
```
Comment (bazzadp) [inline src/content/ja/2019/javascript.md:329]: Missed a translation

File: src/content/ja/2019/javascript.md [hunk 4]
```diff
@@ -0,0 +1,433 @@
...
+   </a>
+	<div id="fig5-description" class="visually-hidden">p10パーセンタイルではデスクトップとモバイルそれぞれ4/4のリクエストを示す棒グラフ、p25では10/9、p50では19/18、p75では33/32、p90では53/52が使用されています。</div>
+	<figcaption id="fig5-caption">図5. 総JavaScriptリクエスト数の分布。</figcaption>
+</figure>
+
+中央値では、デスクトップ用に19件、モバイル用に18件のリクエストが送信されています。
+
+### ファーストパーティ対サードパーティ
+
+これまでに分析した結果のうち、全体のサイズとリクエスト数が考慮されていました。しかし、大多数のウェブサイトでは、取得して使用しているJavaScriptコードのかなりの部分が[サードパーティ](./サードパーティ)のソースから来ています。
```
Comment (bazzadp) [inline src/content/ja/2019/javascript.md:94]:
```suggestion
これまでに分析した結果のうち、全体のサイズとリクエスト数が考慮されていました。しかし、大多数のウェブサイトでは、取得して使用しているJavaScriptコードのかなりの部分が[サードパーティ](./third-parties)のソースから来ています。
```

File: src/content/ja/2019/javascript.md [hunk 5]
```diff
@@ -0,0 +1,433 @@
...
+
+<figure>
+   <a href="/static/images/2019/javascript/fig9.png">
+      <img src="/static/images/2019/javascript/fig9.png" alt="図9. モバイルでダウンロードされたJavaScriptの総ダウンロード数の分布。" aria-labelledby="fig9-caption" aria-describedby="fig9-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vTpzDb9HGbdVvin6YPTOmw11qBVGGysltxmH545fUfnqIThAq878F_b-KxUo65IuXaeFVSnlmJ5K1Dm/pubchart?oid=1611383649&format=interactive">
+   </a>
+	<div id="fig9-description" class="visually-hidden">棒グラフは、p10パーセンタイルではファーストパーティとサードパーティのリクエストでそれぞれ0/17バイトの JavaScriptがモバイルでダウンロードされていることを示していますが、p25では6/54、p50では83/217、p75では189/477、p90では380/827です。</div>
+	<figcaption id="fig9-caption">図9. モバイルでダウンロードされたJavaScriptの総ダウンロード数の分布。</figcaption>
+</figure>
+
+中央値では、モバイルとデスクトップの両方で、開発者が作成したファーストパーティのコードよりもサードパーティのコードの方が89％多く使用されています。これは、サードパーティのコードが肥大化の最大の要因の1つであることを明確に示しています。サードパーティの影響についての詳細は、["サードパーティ"](./third-party)の章を参照してください。
```
Comment (bazzadp) [inline src/content/ja/2019/javascript.md:132]:
```suggestion
中央値では、モバイルとデスクトップの両方で、開発者が作成したファーストパーティのコードよりもサードパーティのコードの方が89％多く使用されています。これは、サードパーティのコードが肥大化の最大の要因の1つであることを明確に示しています。サードパーティの影響についての詳細は、["サードパーティ"](./third-parties)の章を参照してください。
```

File: src/content/ja/2019/javascript.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2019/javascript.md:8]: Thanks, I fixed.

File: src/content/ja/2019/javascript.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2019/javascript.md:283]: Thanks, I fixed.

File: src/content/ja/2019/javascript.md (hunk 4 above)
Comment (ksakae1216) [reply src/content/ja/2019/javascript.md:94]: Sorry, I fixed.

File: src/content/ja/2019/javascript.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2019/javascript.md:329]: Thanks, I fixed.

File: src/content/ja/2019/javascript.md (hunk 5 above)
Comment (ksakae1216) [reply src/content/ja/2019/javascript.md:132]: Sorry, I fixed.

**Pull request merged**